crml/cennzx-spot/src/lib.rs

```diff
@@ -757,14 +752,12 @@ impl<T: Trait> Module<T> {
 		output_reserve: T::Balance,
 		fee_rate: FeeRate<PerMillion>,
 	) -> sp_std::result::Result<T::Balance, DispatchError> {
-		if input_reserve.is_zero() || output_reserve.is_zero() {
+		if input_reserve.is_zero() && output_reserve.is_zero() {
```

Comment:
We can't actually add liquidity to a pool without adding both assets, so don't think the following changes are valid.

Reply:
I'll remove those changes